opencti-platform/opencti-front/src/private/Index.tsx

```diff
@@ -75,8 +76,9 @@ const Index = ({ settings }: IndexProps) => {
   return (
     <>
       <SystemBanners settings={settings} />
-      {(settings?.platform_session_idle_timeout ?? 0) > 0 && <TimeoutLock />}
+      {(settings.platform_session_idle_timeout ?? 0) > 0 && <TimeoutLock />}
       <SettingsMessagesBanner />
+      <PlatformCriticalAlertDialog alerts={settings.platform_critical_alerts}/>
```

Comment:
New popup dialog for admins when the backend has some "critical alerts" to show.
This is for warning about groups with null confidence levels, but could be extended to other types of alerts